provider/pkg/provider/provider.go

```diff
@@ -359,11 +359,13 @@ func (p *googleCloudProvider) Create(ctx context.Context, req *rpc.CreateRequest
 	}, nil
 }
 
+
 func (p *googleCloudProvider) prepareAPIInputs(
-	inputs resource.PropertyMap,
+	inputs resource.PropertyMap, state resource.PropertyMap,
```

Comment:
nit: You can declare args as a comma-separated list if they are the same type, e.g.
```go
inputs, state resource.PropertyMap,
```